### ISSUE TITLE
您好，有一些问题想要请教您，希望您能抽一些时间回复，祝您工作顺利，学术长青！期待您的回复

### ISSUE THREAD

**Issue by lin123596** [tanfy929/DS-Net#1]
我想问一下要运行这套程序需要配置什么环境，像cuda ptorch cudnn这些的版本，我发现这两个链接打不开，想知道这两个链接跟别的链接里面的内容有没有重复：https://drive.google.com/drive/folders/1AVhtSnQotZKJO3Nx4DlEIdZSBfj4KhJ-?usp=drive_link
https://drive.google.com/drive/folders/1AVhtSnQotZKJO3Nx4DlEIdZSBfj4KhJ-?usp=drive_link
祝您工作顺利，学术长青！期待您的回复

**Comment by tanfy929** [comment]
您好，感谢您对本工作的关注。这套程序对环境并没有特殊要求，我们也在很多不同版本的环境下进行了测试。我给您邮件发送了两个.txt文件可做参考。关于Data和checkpoint在此都分别给出了百度网盘链接及Google drive链接，内容相同。谢谢您的提醒，我已经对访问权限进行了更新！后续有问题欢迎与我继续联系：tanfy929@stu.xjtu.edu.cn

**Comment by tanfy929** [comment]
您好，感谢您对本工作的关注。这套程序对环境并没有特殊要求，我们也在很多不同版本的环境下进行了测试。我给您附件发送了两个.txt文件可做参考。关于Data和checkpoint在GitHub中都分别给出了百度网盘链接及Google ***@***.*** / ***@***.***













在 2025-04-27 18:22:53，"lin123596" ***@***.***> 写道：

lin123596 created an issue (tanfy929/DS-Net#1)

我想问一下要运行这套程序需要配置什么环境，像cuda ptorch cudnn这些的版本，我发现这两个链接打不开，想知道这两个链接跟别的链接里面的内容有没有重复：https://drive.google.com/drive/folders/1AVhtSnQotZKJO3Nx4DlEIdZSBfj4KhJ-?usp=drive_link
https://drive.google.com/drive/folders/1AVhtSnQotZKJO3Nx4DlEIdZSBfj4KhJ-?usp=drive_link
祝您工作顺利，学术长青！期待您的回复

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
# This file may be used to create an environment using:
# $ conda create --name <env> --file <this file>
# platform: linux-64
_libgcc_mutex=0.1=main
_openmp_mutex=4.5=1_gnu
absl-py=1.0.0=pypi_0
asttokens=2.0.5=pypi_0
backcall=0.2.0=pypi_0
blas=1.0=mkl
blosc=1.21.0=h8c45485_0
brotli=1.0.9=he6710b0_2
brotlipy=0.7.0=py39h27cfd23_1003
brunsli=0.1=h2531618_0
bzip2=1.0.8=h7b6447c_0
c-ares=1.18.1=h7f8727e_0
ca-certificates=2022.9.24=ha878542_0
cachetools=5.0.0=pypi_0
certifi=2022.9.24=pyhd8ed1ab_0
cffi=1.15.0=py39hd667e15_1
cfitsio=3.470=hf0d0db6_6
charls=2.2.0=h2531618_0
charset-normalizer=2.0.4=pyhd3eb1b0_0
cloudpickle=2.0.0=pyhd3eb1b0_0
conda-content-trust=0.1.1=pyhd3eb1b0_0
conda-package-handling=1.7.3=py39h27cfd23_1
cryptography=36.0.0=py39h9ce1e76_0
cudatoolkit=11.3.1=h2bc3f7f_2
cycler=0.11.0=pypi_0
cytoolz=0.11.0=py39h27cfd23_0
dask-core=2022.5.0=py39h06a4308_0
decorator=5.1.1=pypi_0
einops=0.4.1=pypi_0
executing=0.8.3=pypi_0
ffmpeg=4.3=hf484d3e_0
fonttools=4.33.3=pypi_0
freetype=2.11.0=h70c0345_0
fsspec=2022.3.0=py39h06a4308_0
giflib=5.2.1=h7b6447c_0
gmp=6.2.1=h2531618_2
gnutls=3.6.15=he1e5248_0
google-auth=2.6.6=pypi_0
google-auth-oauthlib=0.4.6=pypi_0
grpcio=1.46.1=pypi_0
idna=3.3=pyhd3eb1b0_0
imagecodecs=2021.8.26=py39h4cda21f_0
imageio=2.9.0=pyhd3eb1b0_0
importlib-metadata=4.11.3=pypi_0
intel-openmp=2021.4.0=h06a4308_3561
ipython=8.3.0=pypi_0
jedi=0.18.1=pypi_0
joblib=1.1.1=py39h06a4308_0
jpeg=9e=h7f8727e_0
jxrlib=1.1=h7b6447c_2
kiwisolver=1.4.2=pypi_0
krb5=1.19.2=hac12032_0
lame=3.100=h7b6447c_0
lcms2=2.12=h3be6417_0
ld_impl_linux-64=2.35.1=h7274673_9
lerc=3.0=h295c915_0
libaec=1.0.4=he6710b0_1
libcurl=7.82.0=h0b77cf5_0
libdeflate=1.8=h7f8727e_5
libedit=3.1.20210910=h7f8727e_0
libev=4.33=h7f8727e_1
libffi=3.3=he6710b0_2
libgcc-ng=9.3.0=h5101ec6_17
libgfortran-ng=7.5.0=ha8ba4b0_17
libgfortran4=7.5.0=ha8ba4b0_17
libgomp=9.3.0=h5101ec6_17
libiconv=1.16=h7f8727e_2
libidn2=2.3.2=h7f8727e_0
libnghttp2=1.46.0=hce63b2e_0
libpng=1.6.37=hbc83047_0
libssh2=1.10.0=h8f2d780_0
libstdcxx-ng=9.3.0=hd4cf53a_17
libtasn1=4.16.0=h27cfd23_0
libtiff=4.2.0=h85742a9_0
libunistring=0.9.10=h27cfd23_0
libuv=1.40.0=h7b6447c_0
libwebp=1.2.2=h55f646e_0
libwebp-base=1.2.2=h7f8727e_0
libzopfli=1.0.3=he6710b0_0
lmdb=1.3.0=pypi_0
locket=1.0.0=py39h06a4308_0
lz4-c=1.9.3=h295c915_1
markdown=3.3.7=pypi_0
matplotlib=3.5.2=pypi_0
matplotlib-inline=0.1.3=pypi_0
mkl=2021.4.0=h06a4308_640
mkl-service=2.4.0=py39h7f8727e_0
mkl_fft=1.3.1=py39hd3c417c_0
mkl_random=1.2.2=py39h51133e4_0
ncurses=6.3=h7f8727e_2
nettle=3.7.3=hbbd107a_1
networkx=2.7.1=pyhd3eb1b0_0
nibabel=3.2.1=pypi_0
numpy=1.21.5=py39he7a7128_2
numpy-base=1.21.5=py39hf524024_2
oauthlib=3.2.0=pypi_0
opencv-python-headless=4.5.5.64=pypi_0
openh264=2.1.1=h4ff587b_0
openjpeg=2.4.0=h3ad879b_0
openssl=1.1.1q=h7f8727e_0
packaging=21.3=pyhd3eb1b0_0
parso=0.8.3=pypi_0
partd=1.2.0=pyhd3eb1b0_1
pexpect=4.8.0=pypi_0
pickleshare=0.7.5=pypi_0
pillow=9.0.1=py39h22f2fdc_0
pip=21.2.4=py39h06a4308_0
prompt-toolkit=3.0.29=pypi_0
protobuf=3.20.1=pypi_0
ptyprocess=0.7.0=pypi_0
pure-eval=0.2.2=pypi_0
pyasn1=0.4.8=pypi_0
pyasn1-modules=0.2.8=pypi_0
pycosat=0.6.3=py39h27cfd23_0
pycparser=2.21=pyhd3eb1b0_0
pygments=2.12.0=pypi_0
pynvml=11.4.1=pyhd8ed1ab_0
pyopenssl=21.0.0=pyhd3eb1b0_1
pyparsing=3.0.9=py39h06a4308_0
pysocks=1.7.1=py39h06a4308_0
python=3.9.7=h12debd9_1
python-dateutil=2.8.2=pypi_0
pytorch=1.10.1=py3.9_cuda11.3_cudnn8.2.0_0
pytorch-mutex=1.0=cuda
pywavelets=1.3.0=py39h7f8727e_0
pyyaml=6.0=py39h7f8727e_1
readline=8.1.2=h7f8727e_1
requests=2.27.1=pyhd3eb1b0_0
requests-oauthlib=1.3.1=pypi_0
rsa=4.8=pypi_0
ruamel_yaml=0.15.100=py39h27cfd23_0
scikit-image=0.19.2=py39h51133e4_0
scikit-learn=1.0.2=py39h51133e4_1
scipy=1.8.0=pypi_0
setuptools=58.0.4=py39h06a4308_0
six=1.16.0=pyhd3eb1b0_0
snappy=1.1.9=h295c915_0
sqlite=3.37.0=hc218d9a_0
stack-data=0.2.0=pypi_0
tensorboard=2.9.0=pypi_0
tensorboard-data-server=0.6.1=pypi_0
tensorboard-plugin-wit=1.8.1=pypi_0
threadpoolctl=2.2.0=pyh0d69192_0
tifffile=2021.7.2=pyhd3eb1b0_2
tk=8.6.11=h1ccaba5_0
toolz=0.11.2=pyhd3eb1b0_0
torchaudio=0.10.1=py39_cu113
torchvision=0.11.2=py39_cu113
tqdm=4.62.3=pyhd3eb1b0_1
traitlets=5.2.0=pypi_0
typing_extensions=4.1.1=pyh06a4308_0
tzdata=2021e=hda174b7_0
urllib3=1.26.7=pyhd3eb1b0_0
wcwidth=0.2.5=pypi_0
werkzeug=2.1.2=pypi_0
wheel=0.37.1=pyhd3eb1b0_0
xz=5.2.5=h7b6447c_0
yaml=0.2.5=h7b6447c_0
zfp=0.5.5=h295c915_6
zipp=3.8.0=pypi_0
zlib=1.2.11=h7f8727e_4
zstd=1.4.9=haebb681_0

absl-py==1.0.0
asttokens==2.0.5
backcall==0.2.0
brotlipy==0.7.0
cachetools==5.0.0
certifi==2022.9.24
cffi @ file:///opt/conda/conda-bld/cffi_1642701102775/work
charset-normalizer @ file:///tmp/build/80754af9/charset-normalizer_1630003229654/work
cloudpickle @ file:///tmp/build/80754af9/cloudpickle_1632508026186/work
conda-content-trust @ file:///tmp/build/80754af9/conda-content-trust_1617045594566/work
conda-package-handling @ file:///tmp/build/80754af9/conda-package-handling_1618262147379/work
cryptography @ file:///tmp/build/80754af9/cryptography_1639414572950/work
cycler==0.11.0
cytoolz==0.11.0
dask @ file:///opt/conda/conda-bld/dask-core_1653082102769/work
decorator==5.1.1
einops==0.4.1
executing==0.8.3
fonttools==4.33.3
fsspec @ file:///opt/conda/conda-bld/fsspec_1652802105984/work
google-auth==2.6.6
google-auth-oauthlib==0.4.6
grpcio==1.46.1
idna @ file:///tmp/build/80754af9/idna_1637925883363/work
imagecodecs @ file:///tmp/build/80754af9/imagecodecs_1635529108216/work
imageio @ file:///tmp/build/80754af9/imageio_1617700267927/work
importlib-metadata==4.11.3
ipython==8.3.0
jedi==0.18.1
joblib @ file:///croot/joblib_1666298844297/work
kiwisolver==1.4.2
lmdb==1.3.0
locket @ file:///opt/conda/conda-bld/locket_1652903118915/work
Markdown==3.3.7
matplotlib==3.5.2
matplotlib-inline==0.1.3
mkl-fft==1.3.1
mkl-random @ file:///tmp/build/80754af9/mkl_random_1626186066731/work
mkl-service==2.4.0
networkx @ file:///opt/conda/conda-bld/networkx_1647437648384/work
nibabel==3.2.1
numpy @ file:///opt/conda/conda-bld/numpy_and_numpy_base_1651563629415/work
oauthlib==3.2.0
opencv-python-headless==4.5.5.64
packaging @ file:///tmp/build/80754af9/packaging_1637314298585/work
parso==0.8.3
partd @ file:///opt/conda/conda-bld/partd_1647245470509/work
pexpect==4.8.0
pickleshare==0.7.5
Pillow==9.0.1
prompt-toolkit==3.0.29
protobuf==3.20.1
ptyprocess==0.7.0
pure-eval==0.2.2
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycosat==0.6.3
pycparser @ file:///tmp/build/80754af9/pycparser_1636541352034/work
Pygments==2.12.0
pynvml @ file:///home/conda/feedstock_root/build_artifacts/pynvml_1639061605391/work
pyOpenSSL @ file:///tmp/build/80754af9/pyopenssl_1635333100036/work
pyparsing @ file:///opt/conda/conda-bld/pyparsing_1661452539315/work
PySocks @ file:///tmp/build/80754af9/pysocks_1605305812635/work
python-dateutil==2.8.2
PyWavelets @ file:///tmp/build/80754af9/pywavelets_1648710015787/work
PyYAML==6.0
requests @ file:///opt/conda/conda-bld/requests_1641824580448/work
requests-oauthlib==1.3.1
rsa==4.8
ruamel-yaml-conda @ file:///tmp/build/80754af9/ruamel_yaml_1616016711199/work
scikit-image @ file:///tmp/build/80754af9/scikit-image_1648214171611/work
scikit-learn @ file:///tmp/build/80754af9/scikit-learn_1642617106979/work
scipy==1.8.0
six @ file:///tmp/build/80754af9/six_1623709665295/work
stack-data==0.2.0
tensorboard==2.9.0
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
threadpoolctl @ file:///Users/ktietz/demo/mc3/conda-bld/threadpoolctl_1629802263681/work
tifffile @ file:///tmp/build/80754af9/tifffile_1627275862826/work
toolz @ file:///tmp/build/80754af9/toolz_1636545406491/work
torch==1.10.1
torchaudio==0.10.1
torchvision==0.11.2
tqdm @ file:///tmp/build/80754af9/tqdm_1635330843403/work
traitlets==5.2.0
typing_extensions @ file:///opt/conda/conda-bld/typing_extensions_1647553014482/work
urllib3==1.26.7
wcwidth==0.2.5
Werkzeug==2.1.2
zipp==3.8.0
